### PR TITLE
fixup dead links discovered by Markdown links Check

### PR DESCRIPTION
**Pull request by pheiduck**
Error noticeable in Markdown links CI: 
ERROR: 1 dead links found!   [✖] https://msdn.microsoft.com/en-us/library/f2ccy3wt.aspx → Status: 403

### CODE REVIEW

**Comment by pheiduck** [comment]
https://curl.zuul.vexxhost.dev/builds Zuul Builds are as well not reachable

**Comment by pheiduck** [comment]
I added a mlc_config.json file to ignore https://curl.zuul.vexxhost.dev/

File: winbuild/README.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ Open a Visual Studio Command prompt:
  [type] Native of Cross platform build.  This type of command prompt may not
  exist in all Visual Studio versions.
 
- See also: [Set the Path and Environment Variables for Command-Line Builds](https://msdn.microsoft.com/en-us/library/f2ccy3wt.aspx)
+ See also: [Set the Path and Environment Variables for Command-Line Builds](https://docs.microsoft.com/en-us/cpp/build/building-on-the-command-line?redirectedfrom=MSDN&view=msvc-170)
```
Comment (bagder) [inline winbuild/README.md:59]: I think `https://docs.microsoft.com/en-us/cpp/build/building-on-the-command-line` should work without having to mention direct and explicit view in the URL

File: mlc_config.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "ignorePatterns": [
+    {
+      "pattern": "^https://curl.zuul.vexxhost.dev"
+    }
+  ]
+}
```
Comment (bagder) [inline mlc_config.json:7]: I still hope that the vexxhost.dev related issues (cert expired) are temporary and will be fixed but maybe we need to accept this fact now. The certs expired on March 8, which is now 9 days ago... :cry:

Comment (bagder) [reply mlc_config.json:7]: BTW, does this file support comments? Can we add an explanation here what the file is for using such?

File: winbuild/README.md (hunk 1 above)
Comment (pheiduck) [reply winbuild/README.md]: Done! https://github.com/curl/curl/pull/8597/commits/3a2a6241d815a2d0c3d95e664257f85bc2d049d1

File: mlc_config.json (hunk 2 above)
Comment (pheiduck) [reply mlc_config.json:8]: Done here: https://github.com/curl/curl/pull/8597/commits/d7e9292a8f35cff16f4f163f8c728e799110c675

**Review by bagder — APPROVED** [review]

**Comment by bagder** [comment]
Thanks!

**Review by vszakats — COMMENTED** [review]
Thank you!

**Pull request closed without merging**